database/index.js

```diff
@@ -1,15 +1,15 @@
 const sampleData = require('./reviewsSampleData.js');
 
```

Comment:
Group all const requires together, if many consider breaking into npm module requires and node file requires

Reply:
Done